tests/integration/static_ip_address_test.py

```diff
@@ -113,7 +115,7 @@ def test_edit_static_ipv4_address_and_prefix(setup_eth1_ipv4):
                 'ipv4': {
                     'enabled': True,
                     'address': [
-                        {'ip': IPV4_ADDRESS2, 'prefix-length': 23}
+                        {'ip': IPV4_ADDRESS2, 'prefix-length': 24}
```

Comment:
@tyll this is wrong, the test is checking if the address *and* prefix changed. Please revert back this part.

Reply:
Good catch, I changed the prefix to `30`, `23` would be outside the range of the test network.